index.html

```diff
@@ -2610,6 +2619,161 @@ <h2>Security Considerations</h2>
 
   </section>
 
+  <section class="appendix">
+    <h4>Timecode-related metadata</h4>
+    <p><a>DAPT Documents</a> express time as <em>media time</em>,
+      which assumes that there is a reference start time (zero) on a media
+      timeline, and a fixed playrate.
+      An alternative scheme that is used in some workflows is to
+      synchronise media components using timecode,
+      time stamps that are applied, for example to each frame of video.
```

Comment:
Is "time stamps that are applied" intended? It feels like left over text. Suggest delete it.
```suggestion
      for example to each frame of video.
```